MonoGame.Framework.Content.Pipeline/Graphics/EffectMaterialContent.cs

```diff
@@ -8,19 +8,7 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Graphics
 {
     public class EffectMaterialContent : MaterialContent
     {
-        public const string EffectKey = "Effect";
-        public const string CompiledEffectKey = "CompiledEffect";
```

Comment:
`EffectKey` & and `CompiledEffectKey` are part of XNA API. we can't remove them.
"Specifies the key of the key/data pair for the compiled effect property.
This key/data pair is stored in the OpaqueData property of the material." 

https://msdn.microsoft.com/en-us/library/microsoft.xna.framework.content.pipeline.graphics.effectmaterialcontent.effectkey(v=xnagamestudio.40).aspx
https://msdn.microsoft.com/en-us/library/microsoft.xna.framework.content.pipeline.graphics.effectmaterialcontent.compiledeffectkey(v=xnagamestudio.40).aspx